src/app/home/events/events.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+
+import { Event } from '../../shared/models/event.model';
+import { mockEvents } from '../../shared/constants/mocks';
+
+@Component({
+  selector: 'sw-events',
+  templateUrl: './events.component.html',
+  styleUrls: ['./events.component.scss']
+})
+export class EventsComponent implements OnInit {
+  events$: Event[];
```

Comment:
This is not an observable, do not use observable notation. Change events$ by events